docs/UserGuide.md

```diff
@@ -667,69 +667,74 @@ _Add screenshot here_
 
 <br>
 
-### Listing all patients for today: `view` `--today`
+### Listing all tasks for today: `view` `--today`
 
-Shows a list of all patients with tasks due today.
+You can view the list of tasks that are due today using the `view` command with the special flag `--today`.
 
 Format: `view --today`
 
+Examples:
+
+Let's say you added the following patients and their tasks:
+* **Physiotherapy appointment** for **Alex Yeoh** at 12:00pm on 2022-11-04
+* **Administer insulin dose** for **Charlotte Oliveiro** at 11:45am on 2022-11-04
+
+If today's date is 2022-11-04, `view --today` will display those 2 tasks.
+
+![result for `view --today`](images/viewTodayResult.png)
+_<div align="center"> Patient and task list displayed after running the `view --today` command </div>_
+
 <br>
 
 ### Listing all tasks: `view` `-p` `--all`
 
-Shows a list of all tasks to be completed.
+You can view the list of tasks for all patients using the `view` command with the special flag `--all`.
 
 Format: `view -p --all`
 
 Examples:
 
-Suppose the following patients were added.
+* `view -p --all` will show the list of all patients with tasks along their list of tasks.
```

Comment:
"with tasks along their list of tasks" seems a bit weird, perhaps could be reworded?

Reply:
Yup I reworded it. Hopefully it's a bit better now.